enhancements/installer/infrastructure-apis-custom-trust.md

```diff
@@ -0,0 +1,236 @@
+---
+title: infrastructure-apis-custom-trust
+authors:
+  - "@abhinavdahiya"
+reviewers:
+  - TBD
```

Comment:
@sdodson 
@wking @bparees @deads2k  as you helped with original egress proxy efforts

@mandre @Fedosin since you already had something like this for OpenStack.
@rgolangh @dougsland for RHV self-signed support.
@enxebre @michaelgugino for machine-api changes
@runcom for mahine-config changes